projects/mysadna/src/app/components/contacts/contacts.component.scss

```diff
@@ -11,14 +19,17 @@ ul {
   padding: 0;
 }
 
-.users-container {
-  text-align: center;
-  >.users-list {
+.contacts-container {
+  >.contacts-list {
     display: flex;
-    justify-content: space-evenly;
+    justify-content: flex-start;
+    @media(max-width: 575.98px) {
+      flex-wrap: wrap;
```

Comment:
It doesn't look right with some sizes:
https://i.imgur.com/FnXlo1j.png
https://i.imgur.com/i20XlDN.png

Can we do something with that?

Reply:
I'm really starting to resent this project only because of the styling.
It's **impossible** to make it look good on all screens, nevermind **similar** to the mockups.
Why do I have to waste so many of my working hours on something that won't end up the it should anyway?

Reply:
We'll make it look good :wink: